CMakeLists.txt

```
@@ -67,10 +67,12 @@ endif()
 # -------------------
 # Google Test のために、ヘッダファイルのあるディレクトリを指定する？
 include_directories(${PROJECT_SOURCE_DIR}/module)
+include_directories(${PROJECT_SOURCE_DIR}/module/API)
 include_directories(${PROJECT_SOURCE_DIR}/test/dummy)
 include_directories(${GTEST_INCLUDE_DIRS} ${PROJECT_SOURCE_DIR}/module)
 file(GLOB SRC_FILES
   ${PROJECT_SOURCE_DIR}/module/*.cpp
+  ${PROJECT_SOURCE_DIR}/module/*/*.cpp
```

意図してないファイルが入るかもしれないから
```suggestion
  ${PROJECT_SOURCE_DIR}/module/API/*.cpp
```
の方が安心かな